src/utils/trust/legal.entity.beneficial.owner.mapper.ts

```diff
@@ -33,8 +41,10 @@ const mapLegalEntityToSession = (
     identification_legal_authority: formData.governingLaw,
     identification_legal_form: formData.legalForm,
     is_service_address_same_as_principal_address: (formData.is_service_address_same_as_principal_address) ? Number(formData.is_service_address_same_as_principal_address) : 0,
-
     is_on_register_in_country_formed_in: (formData.is_on_register_in_country_formed_in) ? Number(formData.is_on_register_in_country_formed_in) : 0,
+    ceased_date_day: formData.stillInvolved === "0" ? formData.ceasedDateDay : "",
```

Comment:
I was expecting to see a mapping for the 'still involved' field here too. Has that moved?

Reply:
Line 68

Reply:
Ok. See this is assigned below (I think). Different to implementation for Individuals but works I think so resolving this.